### PR TITLE
Definitive theme UI

### PR DESCRIPTION
**Pull request by OlivPi**

### CODE REVIEW

File: apps/frontend/src/style/theme.ts [hunk 1]
```diff
@@ -31,237 +44,1488 @@ export const themeFontSize = {
...
-
-  selectionBtnBgd: 'transparent',
-  selectionBtnActBgd: 'transparent',
-  selectionBtnColor: themeColor.greyLight,
-  selectionBtnActColor: themeColor.yellow,
-  selectionBtnBgdHover: themeColor.yellow,
-  selectionBtnBorder: '1',
-  selectionBtnBorderColor: themeColor.greyLight,
-  selectionBtnBorderActColor: themeColor.yellow,
+export const transition = 'all .4s ease';
```
Comment (lutangar) [inline apps/frontend/src/style/theme.ts]: I'm not sure to understand why you had to move all of these properties out of theme object, could you explain it to me?

File: apps/frontend/src/style/theme.ts [hunk 2]
```diff
@@ -31,237 +44,1488 @@ export const themeFontSize = {
...
+export const breakPoints = {
+  palm: 588,
+  desk: 768,
+};
+
+// theme is passed to kepler.gl when it's mounted,
+// it is used by styled-components to pass along to
+// all child components
+
+const input = css<InputProps>`
```
Comment (lutangar) [inline apps/frontend/src/style/theme.ts]: These components would deserve their own files.

File: apps/frontend/src/style/theme.ts (hunk 1 above)
Comment (OlivPi) [reply apps/frontend/src/style/theme.ts]: By digging more into the kepler theme to customize it. I realized that I needed more parameters to make some changes related to the request. Thus bringing us closer to the Figma model.

We will be able to be much more precise. Nothing prevents us from subsequently removing options that we do not need.

I find it easier to take full control of Kepler style behaviors.

File: apps/frontend/src/style/theme.ts (hunk 2 above)
Comment (OlivPi) [reply apps/frontend/src/style/theme.ts]: I totally agree, but i would like to do for a several components.

File: apps/frontend/src/style/theme.ts (hunk 1 above)
Comment (lutangar) [reply apps/frontend/src/style/theme.ts]: Alright then !

File: apps/frontend/src/components/forms/LoginForm.tsx [hunk 3]
```diff
@@ -30,6 +26,11 @@ const StyledLoginForm = styled.form`
   }
 `;
 
+const StyleLink = styled(Link)`
```
Comment (lutangar) [inline apps/frontend/src/components/forms/LoginForm.tsx]:
```suggestion
const StyledLink = styled(Link)`
```

File: apps/frontend/src/components/forms/LoginForm.tsx [hunk 4]
```diff
@@ -77,9 +78,9 @@ export function LoginForm() {
       </StyledLabel>
       <StyledLoginInput id="password" type="password" defaultValue="" {...register('password', { required: true })} />
       {errors.password && <FormattedMessage defaultMessage="This field is required" />}
-      <Link to={'/'}>
+      <StyleLink to={'/'}>
```
Comment (lutangar) [inline apps/frontend/src/components/forms/LoginForm.tsx]:
```suggestion
      <StyledLink to={'/'}>
```

File: apps/frontend/src/components/forms/LoginForm.tsx [hunk 5]
```diff
@@ -77,9 +78,9 @@ export function LoginForm() {
...
         <FormattedMessage defaultMessage="J’ai oublié mon mot de passe" />
-      </Link>
+      </StyleLink>
```
Comment (lutangar) [inline apps/frontend/src/components/forms/LoginForm.tsx]:
```suggestion
      </StyledLink>
```

**Review by lutangar — CHANGES_REQUESTED** [review]

**Review by lutangar — APPROVED** [review]
Looks good to me :+1: 
Let's wait for automatic checks to go :green_circle: before merging tho'

**Pull request merged**